.travis.yml

```diff
@@ -2,3 +2,6 @@ language: node_js
 node_js:
   - "0.10"
   - "0.12"
```

Comment:
No we don't want to support thoses Node versions but it is currently useful to test that PR with all thoses versions.